src/BenchmarksServer/Startup.cs

```diff
@@ -214,13 +224,22 @@ private static string CloneRestoreAndBuild(string path, ServerJob job)
 
             AddSourceDependencies(path, benchmarksDir, dirs);
 
-            var benchmarksPath = Path.Combine(path, benchmarksDir, "src", "Benchmarks");
+            // Install latest SDK and runtime
+            // * Use custom install dir to avoid changing the default install,  which is impossible if other processes
+            //   are already using it.  Custom install is located under default install, so deleting the default
+            //   install will delete both.
```

Comment:
Why is it a goal to delete this copy of the `dotnet` folder when the real one is deleted? Suggest instead the usual `Path.GetTempFileName()` approach and clean the folder up when the `BenchmarksServer` process exits. People clear their temporary folder more often than they delete the `dotnet` folder.